gulpfile.js

```diff
@@ -20,7 +20,22 @@ const hbsFiles = ['./src/patternfly/**/*.hbs'];
 const mdFiles = ['./src/patternfly/**/*.md'];
 
 function clean(cb) {
-  ['./dist', './src/icons/PfIcons'].forEach(dir => removeSync(dir));
+  const cleanDirs = [
+    './dist',
+    './src/icons/PfIcons',
+    'cache',
```

Comment:
Shouldn't this be `.cache`? And doesn't `gatsby clean` already remove `.cache` (so we can just remove this line)?

Reply:
Done